lib/ui/views/messenger/bloc/create_talk_contact_form.dart

```diff
@@ -0,0 +1,75 @@
+part of 'providers.dart';
+
+final _createTalkContactFormProvider = NotifierProvider.autoDispose<
+    CreateTalkContactFormNotifier, CreateTalkContactFormState>(
+  () {
+    return CreateTalkContactFormNotifier();
+  },
+);
+
+@freezed
+class CreateTalkContactFormState with _$CreateTalkContactFormState {
+  const factory CreateTalkContactFormState({
+    required String name,
+    required List<AccessRecipient> members,
+  }) = _CreateTalkContactFormState;
+  const CreateTalkContactFormState._();
+}
+
+class CreateTalkContactFormNotifier
+    extends AutoDisposeNotifier<CreateTalkContactFormState> {
+  CreateTalkContactFormNotifier();
+
+  @override
+  CreateTalkContactFormState build() => const CreateTalkContactFormState(
+        name: '',
```

Comment:
It's better to add a defaultValue in the state instead this initialization

Reply:
It's impossible with a factory constructor, we will have compiling errors if we add default parameters on it

Reply:
```
@freezed
class CreateTalkContactFormState with _$CreateTalkContactFormState {
  const factory CreateTalkContactFormState({
    @Default('') String name,
    @Default([]) List<AccessRecipient> members,
  }) = _CreateTalkContactFormState;
  const CreateTalkContactFormState._();
}

class CreateTalkContactFormNotifier
    extends AutoDisposeNotifier<CreateTalkContactFormState> {
  CreateTalkContactFormNotifier();

  @override
  CreateTalkContactFormState build() => const CreateTalkContactFormState();
```

Reply:
Very nice !!